src/main/include/controllers/DrivetrainController.hpp

```diff
@@ -48,44 +48,30 @@ namespace frc3512 {
 class DrivetrainController : public ControllerBase<7, 2, 4> {
 public:
     /// The wheel radius.
-    static constexpr units::meter_t kWheelRadius = 3.05_in;
+    static constexpr units::meter_t kWheelRadius = 3_in;
 
     /// The drivetrain gear ratio from the encoder to the wheel.
-    static constexpr double kDriveGearRatio = 1.0 / 1.0;
+    static constexpr double kDriveGearRatio = 5.818181 / 1.6;
 
     /// Drivetrain distance per encoder pulse.
     static constexpr double kDpP =
         (2.0 * wpi::numbers::pi * kWheelRadius.value()) * kDriveGearRatio /
         2048.0;
 
     /// Drivetrain chassis width.
-    static constexpr units::meter_t kWidth = [] {
-        auto absoluteValue = [](auto arg) {
-            return arg > decltype(arg){0} ? arg : -1.0 * arg;
-        };
-
-        // These values were collected by rotating the robot in place and
-        // recording the encoder position and gyro heading measurements.
-        // Difference is final measurement minus initial measurement.
-        constexpr auto kLeftPosition = 2.18274_m - 0_m;
-        constexpr auto kRightPosition = (-1.0 * 2.19665_m) - 0_m;
-        constexpr auto kHeading = (-1.0 * 3.1219_rad) - 3.1415_rad;
-
-        return (absoluteValue(kLeftPosition) + absoluteValue(kRightPosition)) /
-               absoluteValue(kHeading) * 1_rad;
-    }();
+    static constexpr units::meter_t kWidth = 3.556_m;
```

Comment:
3.556 m is 11.6 ft, which seems way too wide for a robot drivetrain.

Reply:
Yes, I converted it into meters.